drivers/storage/portworx/portworx_test.go

```diff
@@ -94,15 +94,28 @@ func TestGetStorkEnvList(t *testing.T) {
 			Name:      "px-cluster",
 			Namespace: "kube-test",
 		},
+		Spec: corev1alpha1.StorageClusterSpec{
+			Security: &corev1alpha1.SecuritySpec{
+				Enabled: true,
+			},
+		},
 	}
 
+	cluster.Spec.Image = "portworx/image:2.6.0"
 	envVars := driver.GetStorkEnvList(cluster)
-
-	require.Len(t, envVars, 2)
+	require.Len(t, envVars, 3)
 	require.Equal(t, pxutil.EnvKeyPortworxNamespace, envVars[0].Name)
 	require.Equal(t, cluster.Namespace, envVars[0].Value)
 	require.Equal(t, pxutil.EnvKeyPortworxServiceName, envVars[1].Name)
 	require.Equal(t, component.PxAPIServiceName, envVars[1].Value)
+	require.Equal(t, pxutil.SecurityPXSystemSecretsSecretName, envVars[2].ValueFrom.SecretKeyRef.Name)
+	require.Equal(t, pxutil.SecurityAppsSecretKey, envVars[2].ValueFrom.SecretKeyRef.Key)
+
+	cluster.Spec.Image = "portworx/image:2.5.0"
+	envVars = driver.GetStorkEnvList(cluster)
+	require.Equal(t, pxutil.EnvKeyStorkPXSharedSecret, envVars[2].Name)
+	require.Equal(t, pxutil.SecurityPXSystemSecretsSecretName, envVars[2].ValueFrom.SecretKeyRef.Name)
+	require.Equal(t, pxutil.SecurityAppsSecretKey, envVars[2].ValueFrom.SecretKeyRef.Key)
```

Comment:
Can we add a quick test to see only two envs are passed (without stork shared secret) when security is disabled?